http/listAccounts.go

```diff
@@ -4,22 +4,21 @@ import (
 	"encoding/json"
 	"log"
 	"net/http"
-	"time"
 
 	"github.com/Vivi3008/apiTestGolang/domain"
 	"github.com/gorilla/mux"
 )
 
 type ListAccountResponse struct {
-	Id        string    `json:"id"`
-	Name      string    `json:"name"`
-	Cpf       int       `json:"cpf"`
-	Balance   float64   `json:"balance"`
-	CreatedAt time.Time `json:"createdAt"`
+	Id        string `json:"id"`
+	Name      string `json:"name"`
+	Cpf       int    `json:"cpf"`
+	Balance   int64  `json:"balance"`
+	CreatedAt string `json:"createdAt"`
```

Comment:
O go já transforma o time.Time para o formato RFC339 quando é transformado em json.
```suggestion
	CreatedAt time.Time `json:"createdAt"`
```